chacra/tests/controllers/repos/test_distros.py

```diff
@@ -92,9 +92,8 @@ def test_distro_has_no_built_repos(self, session):
             sha1="head",
         )
         session.commit()
-        result = session.app.get('/repos/foobar/firefly/head/ubuntu/',
-                                 expect_errors=True)
-        assert result.status_int == 404
+        result = session.app.get('/repos/foobar/firefly/head/ubuntu/')
+        assert result.status_int == 200
```

Comment:
this test is now a bit odd: if it is testing that there aren't any built repos why would it expect a 200? (I know this is the side effect of the changes made, but still).

If this test doesn't make sense anymore it might be better to just get rid of it.